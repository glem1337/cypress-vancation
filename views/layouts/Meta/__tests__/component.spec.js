import { shallow } from 'enzyme';

import MetaComponent from '../component';

const mockedHookData = {
  openGraph: {
    url: null,
    title: null,
    type: null,
    image: null,
    siteName: null,
    description: null,
  },
  twitter: {
    card: null,
    image: null,
  },
};
jest.mock('../hook', () => jest.fn(() => mockedHookData));

describe('Meta component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<MetaComponent />);

    expect(wrapper).toMatchSnapshot();
  });
});
