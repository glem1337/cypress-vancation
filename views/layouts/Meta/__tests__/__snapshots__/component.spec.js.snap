// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meta component matches snapshot 1`] = `
<Head>
  <meta
    content="width=device-width, initial-scale=1, shrink-to-fit=no maximum-scale=1.0 user-scalable=no"
    name="viewport"
  />
  <meta
    content="ie=edge"
    httpEquiv="x-ua-compatible"
  />
  <meta
    charSet="utf-8"
  />
  <meta
    content={null}
    property="og:url"
  />
  <meta
    content={null}
    property="og:title"
  />
  <meta
    content={null}
    property="og:type"
  />
  <meta
    content={null}
    property="og:image"
  />
  <meta
    content={null}
    property="og:site_name"
  />
  <meta
    content={null}
    property="og:description"
  />
  <meta
    content="test-key-facebook"
    property="fb:app_id"
  />
  <meta
    content={null}
    property="twitter:card"
  />
  <meta
    content={null}
    name="twitter:image"
  />
  <link
    href="https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl.css"
    rel="stylesheet"
  />
</Head>
`;
